### DISCUSSION TITLE
Error stating that there are no exposed ports even though ive used --http-probe=false

### DISCUSSION THREAD

**Discussion by MikaRanAway** [slimtoolkit/slim#662] [Q&A]
![image](https://github.com/slimtoolkit/slim/assets/128365923/f017ae35-6dc4-415b-b130-11486f94d408)
Ive got no idea how im supposed to go around this

**Comment by kcq** [comment]
@MikaRanAway try specifying --http-probe-off before your image name or try using the --target flag to specify your image name. The current version of the urfave CLI libraries is more strict and it doesn't always pick up the flags if they are placed where it doesn't expect them.
